### PR TITLE
feature/section7 landing page

### PR DESCRIPTION
**Pull request by Megan-Sen**

### CODE REVIEW

File: client/src/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="UTF-8" />
-		<title>Starter Kit</title>
+		<title>Section7</title>
```
Comment (youssef910) [inline client/src/index.html:5]: should be changed to site title, not the task title

File: client/src/pages/Home/WeSupportYou/WeSupportYou.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import "./WeSupportYou.css";
+import section7Image from "https://i.postimg.cc/Xvq2SPBx/supporting-elderly-woman-2021-09-24-03-07-01-utc.jpg";
```
Comment (youssef910) [inline client/src/pages/Home/WeSupportYou/WeSupportYou.js:3]: link imported and never used .. delete

File: client/src/pages/Home/WeSupportYou/dummyWeSupportYou.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (youssef910) [inline client/src/pages/Home/WeSupportYou/dummyWeSupportYou.html:1]: why you need to add html ---- you done the same work on js file

File: client/src/pages/Home/Footer/Footer.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import React from "react";
```
Comment (youssef910) [inline client/src/pages/Home/Footer/Footer.js:1]: you should do the footer in a separate branch and PR

**Review by youssef910 — CHANGES_REQUESTED** [review]

**Review by filkovsp — CHANGES_REQUESTED** [review]

**Review by filkovsp — APPROVED** [review]

**Review by filkovsp — APPROVED** [review]

**Review by filkovsp — APPROVED** [review]
looks done now

**Review by filkovsp — APPROVED** [review]
done as of the latest commit

**Review by filkovsp — APPROVED** [review]

**Review by filkovsp — APPROVED** [review]
done as of the latest commit

**Review by filkovsp — APPROVED** [review]
done as of the latest commit

**Review by filkovsp — APPROVED** [review]
done as of the latest commit

**Review by filkovsp — APPROVED** [review]
done as of the latest commit

**Review by filkovsp — APPROVED** [review]
done as of the latest commit

**Pull request merged**